### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by JunTaoLuo**
## Description

Resolves https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16228. 

### Broken or missing links
These are links in our documentation that are missing documentation or pointing to outdated links. The action items here are to create new documentation, link to updated content or remove the original link.

Top level [Readme](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/efc87a8b61a98e5e5b9824a8da0c4743ef795a3d/README.md)
- [x] 3.vi - should this be pointing to comparing-graphql-output.md?
- [ ] 4.iii - new documentation need to be added
- [x] 5.i - missing architecture overview
- [ ] 5.ii - missing drupal overview, otherwise we should consolidate it and its subsection to Drupal - Memcache
- [ ] 5.iv - missing MetalSmith documentation
- [x] 5.ix - broken link to downstream dependencies

QA [Readme](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/1bcc9937a697280a9669ee1ff84a41ede0abde80/READMES/qa.md)
- [x] Broken Conflence from outdated va-gov domain

Security [Readme](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/efc87a8b61a98e5e5b9824a8da0c4743ef795a3d/READMES/security.md) 
- [x] Fortify Security Scans was removed in https://github.com/department-of-veterans-affairs/va.gov-cms/commit/e9aca3cfa3824481311f7bcd3dc64abca04b590f
- [x] Security policy has been moved to https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/SECURITY.md

### Unlinked or orphaned content
The action items are to decide if these content should be linked in the top level Readme's TOC. Some content may be outdated and should be moved to READMES/historical.

- [x] analytics.md
- [x] blackfire.md
- [x] codeowners.md
- [x] comparing-graphql-output.md
- [x] dependabot-alerts.md
- [ ] distribution-group-mgmt.md
- [x] jsonapi.md
- [ ] multilingual.md
- [x] remove-deprecated-fields.md
- [x] scalability-testing.md
- [x] sentry.md
- [ ] translation.md
- [ ] user_management.md

### Documentation update
The action items here are to update our documentation to reflect our current workflow and recommendations.

- [x] https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/https.md Update OSX instructions for new macOS UI
- [x] Use branches instead of forks in our getting started recommendations
- [x] Update getting started steps to include:
  - create .env from .env.example
  - ddev pull va --skip-files requirement for running locally
- [x] How to find link to cypress logs from CI Tugboat images
- [x] Insert link to the docs at https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms

## Testing done

None since these are doc updates

## Screenshots

None

## QA steps

None

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] ~~Tests have been added if necessary.~~
- [ ] ~~Automated tests have passed.~~
- [ ] ~~Code Quality Tests have passed.~~
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] ~~If there are field changes, front end output has been thoroughly checked.~~

### Select Team for PR review

- [x] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +144,7 @@ This section outlines only the systems utilized by the CMS. See the READMEs in t
 - A single "mirror" environment is regularly populated with a sanitized production database copy.
 - Open Pull Requests get environments created automatically, cloned from the "mirror" environment, with URLs like:
    - [pr123-{hash}.ci.cms.va.gov](https://pr123-{hash}.ci.cms.va.gov) for the CMS
+      - Cypress test logs and artifacts, see [Testing](READMES/testing.md) for details.
```
Comment (ndouglas) [inline README.md:147]: Note to self: I'm moving this file to `automated-tests.md` in a separate PR.

File: READMES/translation.md
Comment (ndouglas) [inline READMES/translation.md:1]: This needs to remain.

File: READMES/multilingual.md
Comment (ndouglas) [inline READMES/multilingual.md:1]: This needs to remain.

File: READMES/devops/distribution-group-mgmt.md
Comment (ndouglas) [inline READMES/devops/distribution-group-mgmt.md:1]: I don't think this should be moved.

File: READMES/devops/user_management.md
Comment (ndouglas) [inline READMES/devops/user_management.md:1]: I don't think this should be moved either. This is mostly helpdesk-facing.

**Review by ndouglas — CHANGES_REQUESTED** [review]
Mostly looks fantastic! Just a couple of moves and a couple of deletions I disagree with. No other major complaints.

File: READMES/translation.md
Comment (JunTaoLuo) [reply READMES/translation.md:1]: Hmm let's also check with @EWashb who recommended moving it to [Product Outlines](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms/product-outlines). Same with multilingual.md

File: README.md (hunk 1 above)
Comment (JunTaoLuo) [reply README.md:147]: I was planning on fixing up these links after you merge your PR.

File: READMES/devops/distribution-group-mgmt.md
Comment (JunTaoLuo) [reply READMES/devops/distribution-group-mgmt.md:1]: Ok I moved it back. Just for my understanding how are this and user_management related to dev work in CMS?

File: READMES/devops/user_management.md
Comment (JunTaoLuo) [reply READMES/devops/user_management.md:1]: I wonder if this is a better home for it then? https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms/helpdesk

File: READMES/translation.md
Comment (JunTaoLuo) [reply READMES/translation.md:1]: Reverted, will follow up in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16498

File: READMES/multilingual.md
Comment (JunTaoLuo) [reply READMES/multilingual.md:1]: Reverted, will follow up in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16498

File: READMES/devops/distribution-group-mgmt.md
Comment (JunTaoLuo) [reply READMES/devops/distribution-group-mgmt.md:1]: Reverted, will follow up in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16498

File: READMES/devops/user_management.md
Comment (JunTaoLuo) [reply READMES/devops/user_management.md:1]: Reverted, will follow up in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16498

**Review by olivereri — APPROVED** [review]

**Comment by JunTaoLuo** [comment]
Since Nate and Edmund are both out and Nate's comments have been addressed (all moved files were reverted), I'm merging this now.

**Pull request merged**